yoti/YotiHelper.php

```diff
@@ -551,13 +551,41 @@ public function createYotiUser($wpUserId, ActivityDetails $activityDetails)
         update_user_meta($wpUserId, 'yoti_user.identifier', $activityDetails->getRememberMeId());
     }
 
+    /**
+     * Get a unique file name.
+     *
+     * @param string $prefix
+     * @param string $extension
+     * @return string
+     */
+    private function getUniqueFilename($prefix, $extension)
```

Comment:
Do you think it would be more appropriate to call this `createUniqueFilename`? `get` implies that we might be simply retrieving it

Reply:
Agree, will change now